src/pages/Home/index.js

```diff
@@ -0,0 +1,534 @@
+  const isActiveIpad = useMediaQuery("(max-width:1300px)");
+  const classes = useStyles();
+  return (
+    <>
+      <CssBaseline />
+      <main>
+        <div className={classes.container}>
+          <Container maxWidth="md">
+            <Typography
+              variant="h4"
+              align={isActive ? "left" : "center"}
+              className={!isActive && classes.heading}
+              color="textPrimary"
+              gutterBottom
+            >
+              Let income not be a barrier to your career dreams
+            </Typography>
+            <Typography
+              variant="h6"
+              align={isActive ? "left" : "center"}
+              gutterBottom
+            >
+              With Meraki, begin your programming journey for free today
+            </Typography>
+            <div>
+              <Grid
+                sx={{ mt: 3 }}
+                container
+                spacing={2}
+                justifyContent="center"
+              >
+                <Grid item>
+                  <Button
+                    className={
+                      isActive ? classes.responsiveBtn : classes.LearningBtn
+                    }
+                    variant="contained"
+                    color="primary"
+                  >
+                    Start Learning
+                  </Button>
+                </Grid>
+                <Grid item>
+                  <Button
+                    className={
+                      isActive ? classes.responsiveBtn : classes.LearningBtn
+                    }
+                    variant="outlined"
+                    color="primary"
+                  >
+                    <img
+                      className={classes.playstoreImg}
+                      src={require("./assets/playstore.svg")}
+                    />
+                    <span className={classes.downloadBtn}>Download Meraki</span>
+                  </Button>
+                </Grid>
+              </Grid>
+            </div>
+          </Container>
+
+          {/* ********************** Section 2 ******************* */}
+
+          <Container sx={{ mt: 4, display: { xs: "none", md: "flex" } }}>
+            <Grid container>
+              <Grid container item xs={4} justifyContent="flex-start">
+                <Card className={classes.typingPopupCard}>
+                  <CardContent>
+                    <Typography color="text.secondary">
+                      I want to be a typing assitant
+                    </Typography>
+                  </CardContent>
+                </Card>
+              </Grid>
+              <Grid item xs={4}></Grid>
+              <Grid container item xs={4} justifyContent="flex-end">
+                <Card className={classes.engineerPopupCard}>
+                  <Typography align="right" color="text.secondary">
+                    I want to be the first software engineer in my family
+                  </Typography>
```

Comment:
Why does the first `Card` have `CardContent` and this one not?
```suggestion
                  <CardContent>
                    <Typography align="right" color="text.secondary">
                      I want to be the first software engineer in my family
                    </Typography>
                  </CardContent>
```